src/config/ServerConfig.ts

```diff
@@ -58,15 +58,21 @@ const defaultServerConfig: ComputedServerConfig = {
   istioAnnotations: {
     istioInjectionAnnotation: 'sidecar.istio.io/inject'
   },
+  istioCanaryRevision: {
+    current: '',
+    upgrade: ''
+  },
   istioIdentityDomain: 'svc.cluster.local',
   istioNamespace: 'istio-system',
   istioLabels: {
     appLabelName: 'app',
     injectionLabelName: 'istio-injection',
+    injectionLabelRev: 'istio.io/rev',
     versionLabelName: 'version'
   },
   kialiFeatureFlags: {
-    istioInjectionAction: true
+    istioInjectionAction: true,
+    istioUpgradeAction: false
```

Comment:
Ah, I put the same comment in the backend :) I see here the default is explicit.
Thanks !